packages/ember/addon/instance-initializers/sentry-performance.ts

```diff
@@ -0,0 +1,74 @@
+import ApplicationInstance from '@ember/application/instance';
+import environmentConfig from 'ember-get-config';
+import Sentry from '@sentry/ember';
+
+export function initialize(appInstance: ApplicationInstance): void {
+  const config = environmentConfig['@sentry/ember'];
+  if (config['disablePerformance']) {
+    return;
+  }
+  instrumentForPerformance(appInstance);
+}
+
+function getTransitionInformation(transition: any, router: any) {
+  const fromRoute = transition?.from?.name;
+  const toRoute = transition ? transition.to.name : router.currentRouteName;
+  return {
+    fromRoute,
+    toRoute,
+  };
+}
+
+export async function instrumentForPerformance(appInstance: ApplicationInstance) {
+  const config = environmentConfig['@sentry/ember'];
+  const sentryConfig = config.sentry;
+  const tracing = await import('@sentry/tracing');
+
+  const router = appInstance.lookup('service:router');
+
+  sentryConfig['integrations'] = [
```

Comment:
This will override `integrations` configured by the end-user

Reply:
Resolved, see https://github.com/getsentry/sentry-javascript/pull/2784/files#diff-bab66c225bed2fcbac494d8725e397a4R109